homeassistant/components/renault/renault_vehicle.py

```diff
@@ -22,21 +22,19 @@
 from .coordinator import RenaultDataUpdateCoordinator
 
 LOGGER = logging.getLogger(__name__)
-_T = TypeVar("_T")
-_P = ParamSpec("_P")
 
 
-def with_error_wrapping(
-    func: Callable[Concatenate[RenaultVehicleProxy, _P], Awaitable[_T]],
-) -> Callable[Concatenate[RenaultVehicleProxy, _P], Coroutine[Any, Any, _T]]:
+def with_error_wrapping[**_P, _R](
+    func: Callable[Concatenate[RenaultVehicleProxy, _P], Awaitable[_R]],
+) -> Callable[Concatenate[RenaultVehicleProxy, _P], Coroutine[Any, Any, _R]]:
     """Catch Renault errors."""
 
     @wraps(func)
     async def wrapper(
         self: RenaultVehicleProxy,
         *args: _P.args,
         **kwargs: _P.kwargs,
-    ) -> _T:
+    ) -> _R:
```

Comment:
It's good practice to name the TypeVar used for the return type `_R` instead of `_T`, just to make it a little more obvious for what it's used.